submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts

```diff
@@ -76,7 +76,7 @@ export class ExperimentHomeComponent implements OnInit {
     this.onSwitchAutoReload();
   }
 
-  fetchExperimentList() {
+  fetchExperimentList(isRefetch: boolean) {
```

Comment:
I think `isAutoReload` is easier to understand.

Reply:
Oh, You're right. I'll modify it ASAP.

Reply:
Oh you replied so fast xd